app/src/main/java/eu/kanade/tachiyomi/util/system/ImageUtil.kt

```diff
@@ -586,9 +557,12 @@ object ImageUtil {
             return false
         }
 
-        Timber.d(
-            "Splitting image with height of $imageHeight into $partCount part with estimated ${optimalSplitHeight}px height per split",
-        )
+        val options = extractImageOptions(imageFile.openInputStream(), resetAfterExtraction = false).apply {
+            inJustDecodeBounds = false
+        }
+        val imageWidth = options.outWidth
```

Comment:
no real need to define this, either below you can do

`val region = Rect(0, splitData.topOffset, options.outWidth, splitData.bottomOffset)`

or

`val region = Rect(0, splitData.topOffset, splitData.splitWidth, splitData.bottomOffset)`

Reply:
Ok